datacollectiondemo/resources/projects.xml

```diff
@@ -291,43 +275,50 @@ To launch this sample application on the Kaa C++ SDK:
 		Click <b>Ok</b> and download the .tar.gz file.
 	</li>
 	<li>
+		Unpack the SDK archive.
+        <pre>
+            tar zxf libs/kaa/kaa* -C libs/kaa
+        </pre>
+	</li>
+	<li>
 		Unpack the downloaded archive and run the following command from the directory containing the unpacked source files.
 		
 		<p>
 			<b>Windows</b>
 			<pre>
-				build.bat deploy
+				mkdir build
+				cd build
+				cmake -G "Nmake Makefiles" ..
+				nmake
 			</pre>
 		</p>
 		<p>
 			<b>Linux, OS X (macOS)</b>
 			<pre>
-				./build.sh deploy
+				mkdir build
```

Comment:
Please re-write the commands the same way it is for 'C SDK' (I somehow used to say 'C SDK' meaning the very first  point these commands appeared in the diff).